test/integration/agent/route_test.go

```diff
@@ -100,7 +100,7 @@ type routeClientOptions struct {
 }
 
 func newTestRouteClient(networkConfig *config.NetworkConfig, options routeClientOptions) (*route.Client, error) {
-	return route.NewClient(networkConfig, options.noSNAT, false, false, false, false, options.nodeSNATRandomFully, false, nil)
+	return route.NewClient(networkConfig, options.noSNAT, false, false, false, false, options.nodeSNATRandomFully, false, nil, 51820)
```

Comment:
maybe use the `WireGuardListenPort` constant from `pkg/apis` to make the code more readable